astraDBApplicationQuickstartPython/README.md

```diff
@@ -1,23 +1,23 @@
 # Overview
 Learn how to connect your app to Astra using Python.
 
-**In this guide, we'll learn how to**
+**In this guide, you'll learn how to**
 - Set up our Python development environment
```

Comment:
Should this be "Set up your Python development environment"?

Reply:
Yes